documentation/translations.md

```diff
@@ -0,0 +1,88 @@
+# Translations
+
+HOF exports [i18n-future](https://github.com/lennym/i18n-future) as 'i18n' which is a tool that supports localization.
+
+Internationalizations are a single JSON file of key/value pairs. They are referenced within the application via dot notation. An example follows:
+
+```
+{
+  "buttons": {
+    "continue": "Continue",
+    "send": "Send",
+    "change": "Change",
+    "close": "Close"
+  }
+}
+```
+
+Where continue would be referenced like so:
+
+```
+buttons.continue
+```
+
+## Setup
+
+i18n is currently used in two areas for hof, within [template mixins](https://github.com/UKHomeOffice/passports-template-mixins) and in the [form wizard](https://github.com/UKHomeOffice/passports-form-wizard).
+
+### Template mixins
+
+To use the localization tool in template mixins it has to be passed in as an additional option.
+
+```
+'use strict';
+
+var hof = require('hof');
+var mixins = hof.mixins;
+var i18nFuture = hof.i18n;
+var router = require('express').Router();
+var path = require('path');
+var _ = require('underscore');
+
+var fields = _.extend({}, require('../common/fields/'), require('./fields/'));
+var i18n = i18nFuture({
+  path: path.resolve(__dirname, './translations/__lng__/__ns__.json')
+});
+
+router.use(mixins(fields, {
+  translate: i18n.translate.bind(i18n)
+}));
+```
+
+This makes the i18n available as a function called `t` within the templates. Translations can be called within the templates like so:
+
+`{{#t}}buttons.continue{{/t}}` which would render `Continue` in the html.
+
+### Wizard usage
+
+To use the localization tool within the wizard it has to be passed in as an additional `translate` option.
+
+```
+'use strict';
+
+var hof = require('hof');
+var wizard = hof.wizard;
+var i18nFuture = hof.i18n;
+var router = require('express').Router();
+var path = require('path');
+var _ = require('underscore');
+
+var i18n = i18nFuture({
+  path: path.resolve(__dirname, './translations/__lng__/__ns__.json')
+});
+
+router.use('/journey-name/', wizard(require('./steps'), fields, {
+  templatePath: path.resolve(__dirname, 'views'),
+  translate: i18n.translate.bind(i18n),
+  params: '/:action?'
+}));
+```
+
+This makes i18n available within the wizard for fields and errors.
+
+
+## Structure
+
+Across hof apps it is recommended you break translations down into sensible chunks to ensure they don't become unweildy. By convention we like translations to live within your journey apps like: `apps/journey-name/translations/src/en` although they can be logically stored anywhere that suits.
+
+We have a tool that combines translation files into a single json for use by i18n called [hof-transpiler](https://github.com/UKHomeOffice/hof-transpiler)
```

Comment:
"If you choose to compartmentalise your locales, use [HOF-transpiler](https://github.com/UKHomeOffice/hof-transpiler) to combine a them into a single file" < Maybe???